### PR TITLE
ZooFooter React Testing Library tests

### PR DESCRIPTION
**Pull request by eatyourgreens**
- add tests for the Zooniverse logotype and tag line.
- delete Enzyme tests from all subcomponents.

## Package
lib-react-components

## How to Review
_Helpful explanations that will make your reviewer happy:_
- What Zooniverse project should my reviewer use to review UX?
- What user actions should my reviewer step through to review this PR?
- Which storybook stories should be reviewed?
- Are there plans for follow up PR’s to further fix this bug or develop this feature?

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
- [ ] Tests are passing locally and on Github
- [ ] Documentation is up to date and changelog has been updated if appropriate
- [ ] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected
- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari (Use Browserstack account as needed)
- [ ] FEM works in a mobile browser

## Refactoring
- [ ] The PR creator has described the reason for refactoring
- [ ] The refactored component(s) continue to work as expected

### CODE REVIEW

File: packages/lib-react-components/src/ZooFooter/ZooFooter.spec.js [hunk 1]
```diff
@@ -5,7 +5,8 @@ import ZooFooter from './ZooFooter'
 
 
 describe('<ZooFooter />', function () {
-  let linkLists, socialLinks, policyNavigation
+  this.timeout(5000)
```
Comment (shaunanoordin) [inline packages/lib-react-components/src/ZooFooter/ZooFooter.spec.js:8]: I assume this is a workaround for the tests timing out randomly.

**Review by shaunanoordin — APPROVED** [review]
## PR Review

Package: `lib-classifier`

This PR updates several tests.

- ZooFooter has a small refactor, and its test updated to be more thorough
- ZooniverseLogotype also has a refactor, which will be of note to #3733 
- Several enzyme tests for the ZooFooter's subcomponents have been deleted
  - ❔ I assume the intent is to replace them with newer RTL tests later?

Looks fine to me.

File: packages/lib-react-components/src/ZooFooter/ZooFooter.spec.js (hunk 1 above)
Comment (eatyourgreens) [reply packages/lib-react-components/src/ZooFooter/ZooFooter.spec.js:8]: User event tests take a while to run, so this to allow plenty of time for those. And the random timeouts too.

**Comment by eatyourgreens** [comment]
All the components with deleted Enzyme tests should be tested here. The coverage report shows good coverage. 

Coverage report for ZooHeader and its subcomponents, showing 100% test coverage.

I’m not 100% sure about deleting the logotype tests, but it does seem that #3733 is duplicating the logotype tests that I’ve added here.

**Pull request merged**